nce/nce_pairwise_sm/args.py

```diff
@@ -20,7 +20,7 @@ def get_args():
     parser.add_argument('--words_dim', type=int, default=50)
     parser.add_argument('--dropout', type=float, default=0.5)
     parser.add_argument('--epoch_decay', type=int, default=15)
-    parser.add_argument('--wordvec_dir', type=str, default='../../../data/word2vec/')
+    parser.add_argument('--wordvec_dir', type=str, default='../../../Castor-data/embeddings/word2vec/')
```

Comment:
different from the path below `word2vec/embeddings` ?

Reply:
Ditto, embeddings are moved to the `embeddings` subdirectory now.